### ISSUE TITLE
WEB SCRAPING

### ISSUE THREAD

**Issue by D3XT3R404** [binbyu/Reader#234]
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

#Dataset_wilayah_IKN_kecamatan
data = {
    "Kecamatan": ["Sepaku", "Loa Kulu", "Samboja", "Loa Janan", "Muara Jawa", "Sanga Sanga"],
    "Luas_Wilayah_km2": [1210, 950, 1000, 870, 600, 500],
    "Tutupan_Hutan_%": [42.31, 38.5, 40.2, 36.7, 30.5, 28.9],
    "Jumlah_Penduduk": [50000, 45000, 60000, 55000, 40000, 30000],
    "Lahan_Terbangun_%": [10, 12, 14, 16, 18, 20],
    "Perkebunan_%": [29.18, 30.0, 27.5, 28.9, 25.6, 22.3],
    "Semak_Belukar_%": [13.74, 15.2, 14.5, 16.0, 17.5, 18.3],
    "Sawah_%": [2.5, 3.0, 2.8, 3.2, 3.5, 3.8],
    "Padang_Rumput_%": [1.2, 1.5, 1.8, 2.0, 2.2, 2.5],
    "Pertambangan_%": [0.5, 0.8, 1.0, 1.2, 1.5, 1.8]
}

#Konversi_DataFrame
df = pd.DataFrame(data)

#Simpan_dataset_CSV
df.to_csv("ikn_data.csv", index=False)

#Visualisasi_Tutupan_Hutan
plt.figure(figsize=(10,5))
sns.barplot(x="Kecamatan", y="Tutupan_Hutan_%", data=df, palette="Greens")
plt.title("Persentase Tutupan Hutan di Wilayah IKN per Kecamatan")
plt.xlabel("Kecamatan")
plt.ylabel("Tutupan Hutan (%)")
plt.show()

#Visualisasi_Korelasi_Heatmap
plt.figure(figsize=(8,6))
sns.heatmap(df.corr(), annot=True, cmap="coolwarm", fmt=".2f")
plt.title("Korelasi Antar Variabel dalam Dataset IKN")
plt.show()

#Data_Training_Deep_Learning
from sklearn.model_selection import train_test_split
from sklearn.preprocessing import StandardScaler

X = df[["Luas_Wilayah_km2", "Jumlah_Penduduk", "Lahan_Terbangun_%", "Perkebunan_%", "Semak_Belukar_%", "Sawah_%", "Padang_Rumput_%", "Pertambangan_%"]]
y = df["Tutupan_Hutan_%"]

#Normalisasi_data
scaler = StandardScaler()
X_scaled = scaler.fit_transform(X)

#Split_data_training_testing
X_train, X_test, y_train, y_test = train_test_split(X_scaled, y, test_size=0.2, random_state=42)
